Notes on variables captured in test closures (#268)
codewars/docs: content/languages/python/codewars-test/index.md

Tests in the Python testing framework are composed of functions decorated with a set of Python decorators. All such functions are automatically discovered and run. The final result for every test block is determined by contained assertions and reported along with its measured execution time.

> Test decorators are created in a way that decorated functions are immediatelly invoked and executed. As a result, they capture and use the current values of variables from enclosing scope.

Reviewer: Sounds confusing.

```suggestion
Test decorators call the functions immediately, so it's safe to use them in a loop.
```

---

> Tests in the Python testing framework are composed of functions decorated with a set of Python decorators.

It's confusing because it's not really "decorating". It's more of a syntax sugar/hack to call the function immediately within a wrapper. I don't think we should go into the details in this document meant for general users, so mentioning this is safe should be enough.
